From 605bd89682aab14635a8fdac3c4e96bc98e6a88f Mon Sep 17 00:00:00 2001
From: Jiri Moskovcak <jmoskovc@redhat.com>
Date: Mon, 7 Jan 2013 13:13:46 +0100
Subject: [PATCH 69/69] fixed the relro flags rhbz#812283

- more info about relro: http://post-office.corp.redhat.com/archives/os-devel-list/2011-July/msg00162.html

Conflicts:
	src/client-python/Makefile.am
	src/report-python/Makefile.am
---
 src/client-python/Makefile.am | 2 +-
 src/report-python/Makefile.am | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/client-python/Makefile.am b/src/client-python/Makefile.am
index f5f8528..4048d56 100644
--- a/src/client-python/Makefile.am
+++ b/src/client-python/Makefile.am
@@ -20,7 +20,7 @@ _reportclient_la_CPPFLAGS = \
     $(PYTHON_CFLAGS) \
     -D_GNU_SOURCE \
     -Wall -Wwrite-strings -Werror \
-    -fPIE -Wl,-z,bind_now
+    -fPIE -Wl,-z,relro,now
 _reportclient_la_LDFLAGS = \
     -module \
     -avoid-version \
diff --git a/src/report-python/Makefile.am b/src/report-python/Makefile.am
index 61af4d2..93ee625 100644
--- a/src/report-python/Makefile.am
+++ b/src/report-python/Makefile.am
@@ -23,8 +23,8 @@ _pyreport_la_CPPFLAGS = \
     $(GLIB_CFLAGS) \
     $(PYTHON_CFLAGS) \
     -D_GNU_SOURCE \
-    -Wall -Wwrite-strings -Werror \
-    -fPIE -Wl,-z,bind_now
+    -Wall -Wwrite-strings -Werror \
+    -fPIE -Wl,-z,relro,now
 _pyreport_la_LDFLAGS = \
     -module \
     -avoid-version \
-- 
1.8.0.1

